### DISCUSSION TITLE
Expo, dotenv and eslint : cannot lint correctly the code

### DISCUSSION THREAD

**Discussion by Nek-** [goatandsheep/react-native-dotenv#338] [Q&A] [answered]
Hello, I'm using eslint but I hit an issue: `error  Unable to resolve path to module '@env'  import/no-unresolved`.

I do not open an issue yet because I saw the process is to discuss it first but I can't get rid of it for a while and I really want to fix this.

Edit: I'm currently using this as a patch in my eslint rules:
```javascript
    'import/no-unresolved': ['error', {
      ignore: ['@env'],
    }],
```

Any idea how to fix it or any lead I could follow for a fix?

BTW it looks super similar to https://github.com/goatandsheep/react-native-dotenv/issues/120 but cleaning the project does not help.

Thanks

**Comment by goatandsheep** [comment, marked as answer]
what are your babel settings?

**Reply by Nek-** [reply]
Everything by default it was not working. We decided to remove the plugin that led to many other issues for us.

**Reply by goatandsheep** [reply]
I'm sorry to hear but I hope you try it for your next project :)

**Reply by Nek-** [reply]
TBH probably not, it feels like it's really not recommended to work with dotenv files. (we did it because we're used to web projects with react/node and/or PHP)

We hired an expert that told us the way to go is to use expo consts... and it seems to work better.

At the end, maybe it's just incompatible with expo but not with react native.

**Reply by goatandsheep** [reply]
To each their own and I'm glad you figured out what worked best for you. I unfortunately haven't been able to replicate any failures using Expo in the latest version, but Expo constants is not a bad alternative!
